Utilities/File.cpp

```diff
@@ -1000,7 +1000,7 @@ fs::file::file(const std::vector<u8> &vec, bs_t<open_mode> mode)
 
 	public:
 		memory_stream(const std::vector<u8> &vec, bs_t<open_mode> mode)
-			: m_vector(vec),
+			: m_vector(std::move(vec)),
```

Comment:
I think it won't do anything unless its just "vec" or a "&&vec", its const so it will have to be copied